app/models/certificate.rb

```diff
@@ -130,6 +130,15 @@ def aia
       end
   end
 
+  def sia
```

Comment:
can we expand this out to `subject_info_access`? This makes me think of [this Sia](https://www.youtube.com/watch/ghPcYqn0p4Y)

Reply:
public key infrastructure software should be so lucky

Much better to make the name obvious and descriptive, changed in 2ec6689

 https://github.com/18F/identity-pki/blob/master/app/models/certificate.rb#L124-L131

Reply:
Didn't realize it was a dupe! Looks like a great opp to refactor into a `parse_to_hash(key_name)` method

Reply:
code climate also disliked the duplication, done in e117fb3